tests/postgres_tests/test_array.py

```diff
@@ -173,12 +173,32 @@ def test_in(self):
             self.objs[:2]
         )
 
+    @unittest.expectedFailure
```

Comment:
Got it. I agree that adding expectedFailure tests is a good approach in this case.